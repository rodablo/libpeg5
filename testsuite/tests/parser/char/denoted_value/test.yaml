driver: python
#input_sources: []
control:
- [SKIP, "True", "TODO: Char is now deprecated."]