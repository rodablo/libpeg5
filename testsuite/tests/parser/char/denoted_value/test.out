CharNode 1:1-1:5 ?
CharNode 1:6-1:10 ?